packages/cli/src/lib/webpack/webpack-client-config.js

```diff
@@ -196,7 +154,7 @@ function prodBuild(config) {
 				preload: 'media',
 				pruneSource: false,
 				logLevel: 'silent',
-				additionalStylesheets: ['route-*.css'],
+				additionalStylesheets: ['routes-*.css'],
```

Comment:
Note: Probably not sound, only used to avoid duping issue w/ Critters (https://github.com/GoogleChromeLabs/critters/issues/88)

Probably need to figure something better out here